scripts/zkai_deepprove_nanozk_adapter_feasibility.py

```
@@ -0,0 +1,370 @@
+        "generated_at",
+        "git_commit",
+        "decision",
+        "question",
+        "adapter_bar",
+        "systems",
+        "systems_commitment",
+        "conclusion",
+        "non_claims",
+    }
+    if set(payload) != expected_fields:
+        raise AdapterFeasibilityError("payload field set mismatch")
```

Ruff (0.15.12)

[warning] Avoid specifying long messages outside the exception class

(TRY003)